datamol/conformers/_conformers.py

```diff
@@ -33,6 +33,9 @@ def generate(
     warning_not_converged: int = 10,
     random_seed: int = 19,
     add_hs: bool = True,
+    fallback_to_random_coords: bool = True,
```

Comment:
If we add this, we also need to add a flag for `enforceChirality`. Instead, we could either:

- Just remove `fallback_to_random_coords` and **do the fallback**.
- Add a `fallback` flag that allow the user to decide whether a fallback is allowed.

Reply:
I initially had both, but as you imply it starts to get a little bit overwhelming. I actually don't think the enforcechirality fallback is likely to solve many failure cases. Given that embed_params allows to disable this, I would alternatively suggest removing this as a fallback option.

Reply:
I don't recall why I used `enforcechirality` (it was a long time ago now) but I guess it's because it was solving some embedding (maybe I was just working on different molecules than you are now).

I am ok to discard it and only keep `fallback_to_random_coords`. Ideally doing that should not modify the tests.